build.sbt

```diff
@@ -276,6 +276,8 @@ def mimaSettings: Seq[Setting[_]] = {
       }.toSet
     },
     mimaBinaryIssueFilters ++= Seq(
+      ProblemFilters.exclude[DirectMissingMethodProblem]
```

Comment:
This is the place where we define the general exclusions for all projects.

Instead, you should add the exclusions on the `persistence-scaladsl` and on `persistence-javadsl` modules and using the right packaging, eg: `scaladsl` and `javadsl`.

I have done it locally and want to push to your branch to move forward with it, but it seems you disable the push from contributor on PR branches.